osmomath/decimal.go

```diff
@@ -332,6 +343,20 @@ func (d BigDec) Quo(d2 BigDec) BigDec {
 	return BigDec{chopped}
 }
 
+// mutative quotient
+func (d BigDec) QuoMut(d2 BigDec) BigDec {
```

Comment:
Can you refactor `Quo` to look like:
https://github.com/osmosis-labs/osmosis/blob/62067092a726b59117345e11f3355e381a72bf3c/osmomath/decimal.go#L270-L276